pkg/patterns/declarative/pkg/manifest/objects.go

```diff
@@ -154,6 +154,20 @@ func (o *Object) MutateContainers(fn func(map[string]interface{}) error) error {
 		return fmt.Errorf("containers was not a list")
 	}
 
+	initContainers, found, err := nestedFieldNoCopy(o.object.Object, "spec", "template", "spec", "initContainers")
+	if err != nil {
+		return fmt.Errorf("error reading containers: %v", err)
```

Comment:
```suggestion
		return fmt.Errorf("error reading init containers: %v", err)
```
Is it more accurate?

Reply:
done.

And I append some unit test too.